scripts/contactUs.js

```diff
@@ -14,13 +15,15 @@ function validate(){
 
 
     // Name can only contain letters a-z in lowercase
+    // Note
```

Comment:
You don't need to define all of these as functions and I think that's what's causing your problem.  You have defined these functions but never actually _call_ them  If you remove the function definition part and have, for example, this:
```
      var nameformat = /^[a-z]+$/;
      if(!userName.match(nameformat))
      {
        errorMessage.innerHTML = "Your name can only contain letters a-z";
      }
```

Everything should work but it will only show a single error at a time even if there are multiple errors on the form.  A better solution would be to add the error message to an array of error messages and then at the very end of the `validate` function loop over the array and add all the messages to the `errorMessage` element.